package.json

```diff
@@ -54,6 +54,7 @@
         "apify-client": "^0.5.25",
         "apify-shared": "^0.1.56",
         "cheerio": "^1.0.0-rc.3",
+        "concat-stream": "^2.0.0",
```

Comment:
Could we use the `readStreamToString` here so that we don't need to include another dependency just to concat stream to a buffer while having to create a function for it anyway?